portal-ui/src/screens/Console/Common/CredentialsPrompt/CredentialsPrompt.tsx

```diff
@@ -194,30 +194,23 @@ const CredentialsPrompt = ({
                 if (consoleCreds) {
                   if (!Array.isArray(consoleCreds)) {
                     consoleExtras = {
-                      console: [
-                        {
                           url: consoleCreds.url,
-                          access_key: consoleCreds.accessKey,
-                          secret_key: consoleCreds.secretKey,
+                          accesskey: consoleCreds.accessKey,
+                          secretkey: consoleCreds.secretKey,
```

Comment:
I think this should be in camelCase

```suggestion
                          accessKey: consoleCreds.accessKey,
                          secretKey: consoleCreds.secretKey,
```